### DISCUSSION TITLE
Populate database from Make (formerly Integromat) service

### DISCUSSION THREAD

**Discussion by slo74design** [supabase/supabase#6408] [Questions] [answered]
Hi,
I'm totally new using Supabase. I discovered it from Youtube videos few days ago.
I have been evaluating three services btw Firebase/Firestore, MongoDB and Supabase to connect to my React project.
I need to include Auth + Database services.

My question: as I have an automation process running on Make (formerly Integromat) (https://www.make.com/en/integrations/postgres), is it possible to connect PostgreSQL database to Integromat to populate it. The connection would be:
- Integromat run a periodic automation to write data inside PostgreSQL on Supabase. 

Is it possible to achieve it?
Thank you
Regards
Salva

**Comment by GaryAustin1** [comment, marked as answer]
The Postgres database with Supabase has all the normal features of Postgres.  You have a database connection with the user postgres available to connect to.
https://supabase.com/docs/guides/database/connecting-to-postgres

**Reply by slo74design** [reply]
Thank you @GaryAustin1, so where could I get all these informations using supabase (requested by Integromat)?
https://drive.google.com/file/d/1Gjx2xXSOGJPNBKH_RF5mxTe_8z3en2oZ/view?usp=sharing

**Reply by GaryAustin1** [reply]
The link I gave you tells you how and where.
Under settings and database.
![image](https://user-images.githubusercontent.com/54564956/162787237-16b067bd-403b-4478-9a64-56233a90dd9e.png)

**Comment by slo74design** [comment]
Perfect, thank you so much for your help @GaryAustin1

**Reply by maxmdev** [reply]
You can use Supabase app for Make.com from here: https://startintegrate.com/apps/it-development/supabase-u8r7vf with module Create a Row.
